src/commands/createNewProject/ProjectCreateStep/PysteinProjectCreateStep.ts

```diff
@@ -0,0 +1,224 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AzExtFsExtra } from '@microsoft/vscode-azext-utils';
+import * as path from 'path';
+import { Progress, Uri, window, workspace } from 'vscode';
+import { hostFileName, localSettingsFileName, ProjectLanguage, pythonFunctionAppFileName, requirementsFileName } from '../../../constants';
+import { IHostJsonV2 } from '../../../funcConfig/host';
+import { IScriptFunctionTemplate } from '../../../templates/script/parseScriptTemplates';
+import { PysteinTemplateProvider } from '../../../templates/script/PysteinTemplateProvider';
+import { confirmOverwriteFile } from '../../../utils/fs';
+import { IProjectWizardContext } from '../IProjectWizardContext';
+import { ScriptProjectCreateStep } from './ScriptProjectCreateStep';
+import { localize } from "../../../localize";
+import { ILocalSettingsJson } from '../../../funcConfig/local.settings';
+import { showMarkdownPreviewFile } from '../../../utils/textUtils';
+
+const gettingStartedFileName = 'getting_started.md';
+
+export class PysteinProjectCreateStep extends ScriptProjectCreateStep {
+    protected gitignore: string = pythonGitignore;
+
+    public async executeCore(context: IProjectWizardContext, progress: Progress<{ message?: string | undefined; increment?: number | undefined }>): Promise<void> {
+        const projectTemplate = await PysteinProjectCreateStep.getProjectTemplate(context);
+
+        const localSettingsContent = projectTemplate.templateFiles[localSettingsFileName];
+
+        if (localSettingsContent) {
+            this.localSettingsJson = JSON.parse(localSettingsContent) as ILocalSettingsJson;
+        } else {
+            // Python V2+ model currently requires the EnableWorkerIndexing feature flag be enabled.
+            this.localSettingsJson.Values ??= {};
+            this.localSettingsJson.Values['AzureWebJobsFeatureFlags'] = 'EnableWorkerIndexing';
+        }
+
+        await super.executeCore(context, progress);
+
+        const files = [
```

Comment:
I'm a bit conflicted about this as the underlying script base type explicitly manages certain files, in useful ways, so blindly pulling template files could unintentionally override that behavior.  (E.g. the PyStein step explicitly pulls certain files out of the template to get them into the form the underlying type requires.) We *could* instead do what you suggest but exclude the files we know are handled differently, but that could still present a problem if the underlying script base type is updated to explicitly handle more files.  In that case, I don't see it as much different than what we're doing now.

Reply:
Yeah, I guess my thought was that we would just trust the templates feed team to serve the correct files with each template. I think that's what we're doing with other Script templates. 

But since we're going to have to change the template provider whenever the templates are being served dynamically anyway, we can always revisit the list of template files then? A static list of files makes sense for a static hard-coded template to me.

Reply:
Ok, you've won me over, I've switch to an "exlusion list" instead of an "inclusion list".